cookbooks/private_dependencies.rst

```diff
@@ -0,0 +1,58 @@
+Private Dependencies
+====================
+
+Some projects make use of private dependencies requiring authentication in order
+for packages to be downloaded. While commiting those credentials in the Git
+repository source code is technically possible, not everyone is comfortable with
+this optional and this is generally considered as a bad practice.
```

Comment:
technically possible, this is considered as a bad practice (remove everything in between)